src/VisualStudio/LiveShare/Impl/AbstractGoToDefinitionWithFindUsagesServiceHandler.cs

```diff
@@ -49,19 +51,18 @@ public async Task<object> HandleAsync(LSP.TextDocumentPositionParams request, Re
             var locations = await GetDefinitionsWithFindUsagesServiceAsync(document, position, cancellationToken).ConfigureAwait(false);
 
             // No definition found - see if we can get metadata as source but that's only applicable for C#\VB.
-            if ((locations.Count == 0) && document.SupportsSemanticModel && this._metadataAsSourceService != null)
+            if ((locations.Count() == 0) && document.SupportsSemanticModel && this._metadataAsSourceService != null)
```

Comment:
ImmutableArray has a `Length` property.

```suggestion
            if ((locations.Length == 0) && document.SupportsSemanticModel && this._metadataAsSourceService != null)
```